power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/dataset.hpp

```diff
@@ -89,7 +214,7 @@ template <dataset_type_tag dataset_type_> class Dataset {
         : meta_data_{&other.meta_data()}, dataset_info_{other.get_description()} {
         for (Idx i{}; i != other.n_components(); ++i) {
             auto const& buffer = other.get_buffer(i);
-            buffers_.push_back(Buffer{.data = buffer.data, .indptr = buffer.indptr});
+            buffers_.push_back(Buffer{buffer.data, buffer.indptr});
```

Comment:
because you have now a constructor, we cannot use the nice named attributes anymore.

Reply:
Reverted to old after removing constructors.